lib/matplotlib/widgets.py

```diff
@@ -514,53 +516,45 @@ class CheckButtons(AxesWidget):
     """
     A GUI neutral set of check buttons.
 
-    For the check buttons to remain responsive you must keep a
-    reference to this object.
+    For the check buttons to remain responsive you must keep a reference to
+    this object.
 
-    The following attributes are exposed
+    Connect to the CheckButtons with the :meth:`on_clicked` method.
 
-     *ax*
-        The :class:`matplotlib.axes.Axes` instance the buttons are
-        located in
+    Attributes
+    ----------
+     ax
+        The :class:`matplotlib.axes.Axes` instance the buttons are located in
 
-     *labels*
+     labels
         List of :class:`matplotlib.text.Text` instances
 
-     *lines*
+     lines
         List of (line1, line2) tuples for the x's in the check boxes.
         These lines exist for each box, but have ``set_visible(False)``
         when its box is not checked.
 
      *rectangles*
         List of :class:`matplotlib.patches.Rectangle` instances
-
-    Connect to the CheckButtons with the :meth:`on_clicked` method
     """
-    def __init__(self, ax, labels, actives=None):
+    def __init__(self, ax, labels, active_buttons):
         """
         Add check buttons to :class:`matplotlib.axes.Axes` instance *ax*
 
         Parameters
         ----------
-        ax : `~matplotlib.axes.Axes`
```

Comment:
`ax` description is lost. The listed parameters must be complete.

Also, I think the original parameter descriptions are better, so the whole docstring should essentially stay as is -- except maybe fot the types, for which we use `list of str` and not `List[str]` according to numpydoc conventions.

Reply:
fixed it, thanks.